File: auto-docs/executables/julia/basic_error_bar.jl
# Learn about API authentication here: {{BASE_URL}}/julia/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

using Plotly

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "x" => [0, 1, 2], 
    "y" => [6, 10, 2], 
    "error_y" => [
      "type" => "data", 
      "array" => [1, 2, 3], 
      "visible" => true
    ], 
    "type" => "scatter"
  ]
]
response = Plotly.plot(data, ["filename" => "basic-error-bar", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/r/basic_error_bar.r
# Learn about API authentication here: plot.ly/r/getting-started
# Find your api_key here: plot.ly/settings/api

library(plotly)
py <- plotly(username='TestBot', key='r1neazxo9w')


data <- list(
  list(
    x = c(0, 1, 2), 
    y = c(6, 10, 2), 
    error_y = list(
      type = "data", 
      array = c(1, 2, 3), 
      visible = TRUE
    ), 
    type = "scatter"
  )
)
response <- py$plotly(data, kwargs=list(filename="basic-error-bar", fileopt="overwrite"))
url <- response$url
